docs/source/depth_estimation.mdx

```diff
@@ -0,0 +1,198 @@
+# Depth estimation
+
+Depth estimation datasets are used to train a model to approximate the relative distance of every pixel in an
+image from the camera aka depth. The applications enabled by these datasets mostly lie in areas like visual machine
+perception and perception in robotics. This guide will show you how to apply transformations
+to a depth estimation dataset.
+
+Before you start, make sure you have up-to-date versions of `albumentations` installed:
+
+```bash
+pip install -U albumentations 
+```
+
+[Albumentations](https://albumentations.ai/) is a Python library for performing data augmentation
+for computer vision. It supports various computer vision tasks such as image classification, object
+detection, segmentation, and keypoint estimation.
+
+This guide uses the [NYU Depth V2](https://huggingface.co/datasets/sayakpaul/nyu_depth_v2) dataset which is 
+comprised of video sequences from a variety of indoor scenes as recorded by both the RGB and Depth cameras
+from the Microsoft [Kinect](http://www.xbox.com/kinect). The dataset consists of scenes from 3 cities and provides images along with
```

Comment:
I don't think it's necessary or relevant to mention here that it was recorded from a Kinect. A link to the dataset card is fine :)

```suggestion
comprised of video sequences from various indoor scenes, recorded by RGB and depth cameras. The dataset consists of scenes from 3 cities and provides images along with
```

Reply:
Agreed. Thanks.